sklearn/ensemble/tests/test_voting_classifier.py

```diff
@@ -428,3 +438,41 @@ def test_transform():
             eclf3.transform(X).swapaxes(0, 1).reshape((4, 6)),
             eclf2.transform(X)
     )
+
+
+def test_prefit_true_calling_fit():
+
+    X = np.array([[-1.1, -1.5], [-1.2, -1.4], [-3.4, -2.2], [1.1, 1.2]])
+    y = np.array([1, 1, 2, 2])
+
+    lr1 = LogisticRegression()
+    lr2 = LogisticRegression()
+    lr1.fit(X, y)
+    lr2.fit(X, y)
+
+    eclf = VotingClassifier(estimators=[('lr1', lr1),
+                                        ('lr2', lr2)],
+                            voting='soft',
+                            prefit=True)
+
+    msg = ('Since `prefit=True`, call `transform`,'
```

Comment:
No, I definitely think we should not *disallow* fitting. What if the VotingClassifier is in a pipeline, as a transformer or as a predictor?

Reply:
Although I suppose there's limited benefit to fitting some previous transformers in the pipeline but them passing the data into pre-fitted predictors.

Hmm...